tests/forcing/test_lisflood.py

```diff
@@ -42,9 +42,6 @@ class MockTaskOutput:
         data_files = (
             create_netcdf('pr', tmp_path / 'lisflood_pr.nc'),
             create_netcdf('tas', tmp_path / 'lisflood_tas.nc'),
-            create_netcdf('e0', tmp_path / 'lisflood_e0.nc'),
```

Comment:
maybe replace them with a `#TODO: add lisvap output files once implemented <issue link>`?

Reply:
a TODO it is added. The lisvap input files are the recipe output.